manuscript/20.supp-figures.md

```diff
@@ -24,14 +24,14 @@
 
-\pagebreak
+![**Model performance relative to the presencce of each state in uncensored lineages.** (a) Visualization of the distribition of cells in the lineage transitioning from state 1 to state 2. (b) Bernoulli parameter approaches the true value when 40-80% of cells are in state 1 and deviates at other percentages for states 1 and 2. (c) Gamma shape parameter approaches the true value then deviates when over 60% of cells are in state 1 for states 1 and 2. (d) Gamma scale parameter approaches the true value consistently for states 1 and 2. (e) State assignment accuracy decreases as the percent of cells in State 1 increases. (f) Errors in estimating the transition probability matrix, T, approaches 0 when 40-75% of cells are in state 1 and increases at other percentages for states 1 and 2. ](./output/figureS04.svg){#fig:prop4}
 
-<!-- (Supp. Figure 5) -->
-### Change in model performance when varying presence of a state for censored population
+<!-- (Supp. Figure 5)
+### Change in model performance when varying presence of a state for censored population -->
 
-![**Performance on populations of censored lineages of increasing number.** (a) Visualization of the proportion of cells in the censored lineage transitioning from state 1 to state 2. (b) Bernoulli parameter deviates from the true value as percent of cells in state 1 increases for state 1 and approaches the true value when the percent of cells in state 1 is above 40% for states 1. (c)-(d) Gamma shape and scale parameters approach the true value then deviates when over 80% of cells are in state 1 for states 1 and 2. (e) State assignment accuracy decreases as the percent of cells in State 1 increases. (f) Errors in estimating the transition probability matrix, T, approaches 0 when 40-75% of cells are in state 1 and increases at other percentages for states 1 and 2.](./output/figureS05.svg){#fig:real_5}
+<!-- ![**Performance on populations of censored lineages of increasing number.** (a) Visualization of the proportion of cells in the censored lineage transitioning from state 1 to state 2. (b) Bernoulli parameter deviates from the true value as percent of cells in state 1 increases for state 1 and approaches the true value when the percent of cells in state 1 is above 40% for states 1. (c)-(d) Gamma shape and scale parameters approach the true value then deviates when over 80% of cells are in state 1 for states 1 and 2. (e) State assignment accuracy decreases as the percent of cells in State 1 increases. (f) Errors in estimating the transition probability matrix, T, approaches 0 when 40-75% of cells are in state 1 and increases at other percentages for states 1 and 2.](./output/figureS05.svg){#fig:real_5} -->
```

Comment:
Is this meant to be commented out?

Reply:
Hmm... can you briefly summarize how each of the three figures are different? I don't want to remove it just because it's weird, but I agree we can reduce these down.

Reply:
Figure 7: tree plot to show censored vs uncensored, and shows only state assignment accuracy for the two cases, simulated for phase-specific emissions.
Figure S04: Shows all the estimation (accuracy, parameter estimation, T and pi errors) for uncensored lineages with simple gamma emissions
Figure S05: Same as S04 only for censored lineages.

Reply:
They all have overlap...

Reply:
Ok, let's get rid of 7 but keep S04 and S05.